java-lib/src/main/java/org/clink/feature/onehotencoder/ClinkOneHotEncoderUtils.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 The Clink Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.clink.feature.onehotencoder;
+
+import org.apache.flink.api.java.tuple.Tuple2;
+import org.apache.flink.ml.feature.onehotencoder.OneHotEncoderModel;
+import org.apache.flink.ml.feature.onehotencoder.OneHotEncoderModelData;
+import org.apache.flink.streaming.api.datastream.DataStream;
+
+import org.apache.commons.collections.IteratorUtils;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Comparator;
+import java.util.List;
+
+/** Utility methods for ClinkOneHotEncoder. */
+public class ClinkOneHotEncoderUtils {
+    /**
+     * Saves the model data of the OneHotEncoderModel to the given path, in format of ProtoBuf.
+     *
+     * @param model The OneHotEncoderModel whose model data is to be saved.
+     * @param path The parent directory of the model data file.
+     */
+    public static void saveModelDataAsProtobuf(OneHotEncoderModel model, String path)
```

Comment:
Would it be better to move the logic here to `ClinkOneHotEncoderModel::save(...)`? And if so, do we still need ClinkOneHotEncoderUtils.java?

Reply:
I'll move this logic to `save()` method and remove `ClinkOneHotEncoderUtils`.